Sources/LocalReceiptParsing/DataConverters/DateFormatter+Extensions.swift

```diff
@@ -45,7 +45,7 @@ extension ISO8601DateFormatter {
             }
 
             func string(from date: Date) -> String {
-                return ISO8601DateFormatter.withMilliseconds.string(from: date)
+                return ISO8601DateFormatter.noMilliseconds.string(from: date)
```

Comment:
JSON requires ISO8601 dates with no milliseconds.
Nothing else uses this, and technically this is more correct as `noMilliseconds` is implemented with only `.withInternetDateTime`.